R/mcmc-diagnostics-nuts.R

```diff
@@ -262,14 +264,16 @@ mcmc_nuts_divergence <- function(x, lp, chain = NULL, ...) {
   violin_lp <- ggplot(violin_lp_data, aes_(x = ~ Value, y = ~ lp)) +
     geom_violin(fill = get_color("l"), color = get_color("lh")) +
     ylab("lp__") +
-    xaxis_title(FALSE)
+    xaxis_title(FALSE) +
+      bayesplot_theme_get()
```

Comment:
I had thought that `bayesplot_theme_get()` would override the change by `xaxis_title()` because `xaxis_title()` can theme xaxis titles. But `xaxis_title(FALSE)` just calls `labs()` so these are all fine.